tests/unit/cluster/cluster-flush-slot.tcl

```diff
@@ -0,0 +1,34 @@
+start_cluster 2 2 {tags {external:skip cluster}} {
+    test "SYNC Flush slot command" {
+        set key_slot [R 0 CLUSTER KEYSLOT FC]
+        set slot_keys_num [R 0 CLUSTER COUNTKEYSINSLOT $key_slot]
+
+        # set key
+        for {set i 0} {$i < 1000} {incr i} {
+            R 0 set "{FC}-$i" "value"
+        }
+        set after_keys_num [expr {$slot_keys_num + 1000}]
+        assert_equal [R 0 CLUSTER COUNTKEYSINSLOT $key_slot] $after_keys_num
+
+        # flush slot key
+        R 0 CLUSTER FLUSHSLOT $key_slot SYNC
+        assert_equal [R 0 CLUSTER COUNTKEYSINSLOT $key_slot] 0
+    }
+
+    test "ASYNC Flush slot command" {
+        set key_slot [R 0 CLUSTER KEYSLOT FC]
+        set slot_keys_num [R 0 CLUSTER COUNTKEYSINSLOT $key_slot]
+
+        # set key
+        for {set i 0} {$i < 1000} {incr i} {
+            R 0 set "{FC}-$i" "value"
+        }
+        set after_keys_num [expr {$slot_keys_num + 1000}]
+        assert_equal [R 0 CLUSTER COUNTKEYSINSLOT $key_slot] $after_keys_num
+
+        # flush slot key
+        R 0 CLUSTER FLUSHSLOT $key_slot ASYNC
+        assert_equal [R 0 CLUSTER COUNTKEYSINSLOT $key_slot] 0
```

Comment:
we should add a wait (`wait_for_condition`). This could be a source of test flakiness.

Reply:
this command execute failed when I rebased to the latest unstable https://github.com/valkey-io/valkey/commit/6df376d68a97e9c0da4549f57db96742b5482202, I try to confirm the cause.